src/infrastructure/oidc/index.js

```diff
@@ -115,7 +115,7 @@ const init = async (app) => {
       }
 
       if(allUserServices && allUserServices.roles){
-        const { roles } = allUserServices;
+        const roles = allUserServices.roles.sort((a, b) => a.name.localeCompare(b.name, 'es', {sensitivity: 'base'}));
```

Comment:
Was this expected changes

Reply:
I think this crept in on the back of a pair programming session with Adam I can remove if you prefer?